api/v2/views/identity.py

```diff
@@ -23,14 +23,14 @@ class Meta:
         fields = ["project_id", "project_uuid", "group_id"]
 
 
-class IdentityViewSet(MultipleFieldLookup, AuthModelViewSet):
+class IdentityViewSet(MultipleFieldLookup, UserListAdminQueryAndUpdate, ModelViewSet):
```

Comment:
Are there other `ViewSet` classes that use this behavior (Users read, admins can update)?

If so, we could package that up similar to what we've done with `AuthModelViewSet`..

Reply:
Yes, but its not clear to me whether custom perm like this are better than having separate views. I think I like the latter more.